code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -204,6 +204,7 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/bonesetter(src)
 	src.modules += new /obj/item/weapon/circular_saw(src)
 	src.modules += new /obj/item/weapon/surgicaldrill(src)
+	src.modules += new /obj/item/weapon/gripper/chemistry(src)

```

Comment:
Surgical borg is underplayed and often SSD, i was actually thinking, if only one can have it, that the surgical is more suitable than crisis. one works in medical, one works outside